manual/gce-sru-19.2.21.txt

```diff
@@ -48,12 +53,16 @@ for release in $SRU_SERIES; do
   ssh "${sshopts[@]}" ubuntu@$VM_IP -- systemd-analyze blame
   ssh "${sshopts[@]}" ubuntu@$VM_IP -- cloud-init analyze show
   ssh "${sshopts[@]}" ubuntu@$VM_IP -- cloud-init analyze blame
+  echo 'Networking config'
+  ssh "${sshopts[@]}" ubuntu@$VM_IP -- cat $netcfg
```

Comment:
Good changes, please promote this common manual script change into sru-templates/gce-sru

Reply:
ok, I'll open a new PR for that (and add the required sudo on the rm path)